tests/code.spec.ts

```diff
@@ -268,8 +268,11 @@ test('press ArrowUp after code block can jump into start of code block', async (
   await page.keyboard.press('Enter');
   await page.keyboard.press('Enter');
   await page.keyboard.press('ArrowUp');
-  const index = await getQuillSelectionIndex(page);
-  expect(index).toBe(0);
+
+  const selectedRectsExist = await page.evaluate(() => {
+    return !!document.querySelector('.affine-page-selected-rects-container');
+  });
+  expect(selectedRectsExist).toEqual(true);
```

Comment:
you can use playwright locator API next time

```ts
const selectedRects = page.locator('.affine-page-selected-rects-container');
await expect(selectedRects).toBeVisible();
await expect(selectedRects).toHaveCount(1);
```

ref https://playwright.dev/docs/api/class-locatorassertions

Reply:
@Himself65 your PR time 😈

Reply:
The `toBeVisible` also will check the element attached and visible. The zero-height container broke the test. I think expecting count is enough in this scene.



> Element is considered visible when it has non-empty bounding box and does not have visibility:hidden computed style. Note that elements of **zero size** or with display:none are not considered visible.


<img width="331" alt="Screenshot 2023-01-09 at 3 30 17 PM" src="https://user-images.githubusercontent.com/18554747/211258947-a7194ee1-9df7-4e35-bf1b-d7daf706c100.png">

Reply:
🐮🍺